spyder/plugins/updatemanager/tests/test_update_manager.py

```diff
@@ -59,6 +59,16 @@ def test_updates_appenv(qtbot, mocker, version, caplog):
         um.start_check_update()
         qtbot.waitUntil(um.update_thread.isFinished)
 
+    if um.update_worker.error:
+        # Possible 403 error - rate limit error, was encountered while doing
+        # the tests
+        # Check error message corresponds to the status code and exit early to
+        # prevent failing the test
+        assert um.update_worker.error == HTTP_ERROR_MSG.format(status_code="403")
+        return
+
+    assert not um.update_worker.error
+
```

Comment:
This should not be an issue now with #22118. @dalthviz, were you still seeing this error after #22118 was merged?

Reply:
I think it's better to leave it to prevent unnecessary errors in our test suite.

Reply:
> @dalthviz, were you still seeing this error after https://github.com/spyder-ide/spyder/pull/22118 was merged?

I think not but since there is a need for the `GITHUB_TOKEN` I though that maybe leaving the workaround could be worthy

Reply:
Thanks @ccordoba12 and @dalthviz. I just wanted to make sure that we weren't still having the rate limit issue after #22118.